nncf/tensorflow/README.md

```diff
@@ -66,14 +67,14 @@ compression_ctrl.export_model("compressed_model.pb", save_format='frozen_graph')
 For a quick start with NNCF-powered compression, you can also try the sample scripts, each of them provides a basic training pipeline for Image Classification and Object Detection correspondingly.
 
 To run the samples please refer to the corresponding tutorials:
-- [Image Classification sample](examples/tensorflow/classification/README.md)
-- [Object Detection sample](examples/tensorflow/object_detection/README.md)
-- [Instance Segmentation sample](examples/tensorflow/segmentation/README.md)
+- [Image Classification sample](/examples/tensorflow/classification/README.md)
```

Comment:
Wrong link target

Reply:
So are you suggesting to use `../examples/tensorflow/classification/README.md` or what? These links work fine in github.

Reply:
@evgeniya-egupova, right, use the `../` format.

Reply:
Done